docs/assets/js/examples.js

```diff
@@ -0,0 +1,263 @@
+---
+---
+$(document).ready(function () {
+
+  const DEFAULT_EXAMPLE_ID = 'terraform-http-basic'
```

Comment:
Should this be in the JS code? Or could we just select the first file tab (which is the first item in `examples.yml`) as the default?